daprdocs/content/en/reference/components-reference/supported-state-stores/setup-mongodb.md

```diff
@@ -67,6 +69,7 @@ If you wish to use MongoDB as an actor store, append the following to the yaml.
 | writeconcern       | N        | The write concern to use | `"majority"`
 | readconcern        | N        | The read concern to use  | `"majority"`, `"local"`,`"available"`, `"linearizable"`, `"snapshot"`
 | operationTimeout   | N        | The timeout for the operation. Defaults to `"5s"` | `"5s"`
+| params             | N        | Additional parameters to use | `"?authSource=daprStore&ssl=true"`
```

Comment:
Can you please provide a paragraph below that describes typical use cases that I would need to apply extra params. Also providing a list of the choices that you can apply today.

Reply:
@greenie-msft Please see the latest changes. Thanks!